File: Strings/pagina05/3088/3088.cpp
#include <iostream>
using namespace std;

string fixText(string text) {
    string fixed = "";
    for (size_t i = 0; i < text.size(); i++) {
        char character = text[i];
        if (i < text.size() - 1) {
            char next = text[i + 1];
            if (character == ' ' && (next == '.' || next == ',')) continue;
        }
        fixed += character;
    }
    return fixed;
}

int main(){
    string input;
    while(getline(cin, input)){
        cout << fixText(input) << endl;
    }
    return 0;
}

File: Strings/pagina05/3088/3088.ts
import * as fs from "fs"
const input = fs.readFileSync("/dev/stdin", "utf8").trim().split("\n")

const fixText = (text: string): string => {
    let fixed: string = "";
    for(let i = 0; i < text.length; i++){
        const char: string = text[i];
        const next: string = text[i + 1];
        if(char === " " && (next === "." || next === ",")){
            continue;
        }
        fixed += char;
    }
    return fixed;
}

console.log(input.map(fixText).join("\n"))